src/Management/src/Endpoint/Actuators/Health/HealthEndpointMiddleware.cs

```diff
@@ -52,20 +52,19 @@ private string GetRequestedHealthGroup(PathString requestPath)
     {
         string[] requestComponents = requestPath.Value?.Split('/') ?? [];
 
-        if (requestComponents.Length > 0)
+        if (requestComponents.Length > 0 && requestComponents[^1] != _endpointOptionsMonitor.CurrentValue.Id)
         {
             return requestComponents[^1];
         }
 
-        _logger.LogWarning("Failed to find anything in the request from which to parse health group name.");
+        _logger.LogTrace("Did not find a health group in the request path.");
 
         return string.Empty;
     }
 
     private bool IsValidGroup(string groupName)
     {
-        return groupName == _endpointOptionsMonitor.CurrentValue.Id || groupName == "{**_}" ||
-            _endpointOptionsMonitor.CurrentValue.Groups.Any(pair => pair.Key == groupName);
+        return string.IsNullOrEmpty(groupName) || groupName == "{**_}" || _endpointOptionsMonitor.CurrentValue.Groups.Any(pair => pair.Key == groupName);
```

Comment:
`"{**_}"` shouldn't be considered a valid group name, looks like a bug.

Reply:
I had removed it earlier before intending to change the group parsing logic, but then a test broke... but now all tests are green without it, so it's an easy delete